### ISSUE TITLE
Ignore Pods directory in git

### ISSUE THREAD

**Issue by keywind** [FacebookUniversity2018/BookTrader#3]
Two many source files in Pods three-party libs, please ignore them.

**Comment by claudiahaddad** [comment]
just making sure, we should delete most of the pod files and folders that arent being used?

**Comment by keywind** [comment]
I don't think this issue is solved, if solved, we won't see the content under Pods directory in Github.

**Comment by claudiahaddad** [comment]
I put BookTrader/Pods in the .gitignore, is there another step? I did pod deintegrate and pod install earlier today but that shouldn't have an effect right?
